content/rs/clusters/monitoring/uptrace-integration.md

```diff
@@ -0,0 +1,185 @@
+---
+Title: Uptrace integration with Redis Enterprise Software
+linkTitle: Uptrace integration
+description: To collect, display, and monitor metrics data from your databases and other cluster components, you can connect Uptrace to your Redis Enterprise cluster using OpenTelemetry Collector.
+weight: 50
+alwaysopen: false
+categories: ["RS"]
+---
+
+To collect, display, and monitor metrics data from your databases and other cluster components, you can connect Uptrace to your Redis Enterprise cluster using OpenTelemetry Collector.
+
+[Uptrace](https://uptrace.dev/get/) is an open-source APM tool that supports distributed tracing, metrics, and logs. You can use it to monitor applications and set up automatic alerts to receive notifications.
+
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) allows to receive, process and export telemetry data in a vendor-agnostic way. You can use it to scrape Prometheus metrics provided by Redis and then export those metrics to Uptrace or any other tool.
+
+You can use Uptrace to:
+
+- Collect and display data metrics not available in the [admin console]({{< relref "/rs/clusters/monitoring/console-metrics-definitions.md" >}}).
+- Get access to pre-built dashboard templates maintained by Uptrace community.
+- Set up automatic alerts and receive notifications via email, Slack, Telegram, and others.
+- Monitor your app performance and logs using [tracing](https://uptrace.dev/opentelemetry/distributed-tracing.html).
+
+![uptrace-redis-nodes](/images/rs/uptrace-redis-nodes.png)
+
+## Installing Collector and Uptrace
+
+Because installing OpenTelemetry Collector and Uptrace can take some time, you can use the [docker-compose](https://github.com/uptrace/uptrace/tree/master/example/redis-enterprise) example that also comes with Redis Enterprise cluster and AlertManager.
+
+Alternatively, you can try the following guides to explore all available options:
+
+<!-- The first link could point to the official Otel doc, but it focuses on installing otelcol instead of otelcol-contrib which is required here -->
+
+- [Getting started with OpenTelemetry Collector](https://uptrace.dev/opentelemetry/collector.html)
+- [Getting started with Uptrace](https://uptrace.dev/get/install.html)
+
+Either way, after installing Uptrace you should have Uptrace UI available at [http://localhost:14318/](http://localhost:14318/).
+
+## Scraping Prometheus metrics
```

Comment:
We also might want to consider changing the **Scrape Prometheus metrics** instructions to a numbered list of steps.
```suggestion
## Scrape Prometheus metrics
```